exercises/concept/stage-heralding/.docs/instructions.md

```diff
@@ -100,7 +100,9 @@ A few things to note here:
 
 - If no organisation is given for a speaker, leave it out in the moderation card.
 - If no time for the Q&A session is given, the speaker opted against answering questions.
-- If the speaker does not specify pronouns, use the speaker's name instead.
+- If the speaker does not specify pronouns, use the speaker's first name instead.
+  Note that in general, it is not safe to assume that the name of a person used in informal contexts is also their first name.
+  However, among the speakers on this stage, this happens to be the case.
```

Comment:
I find it really hard to write this in a way that makes sense since "first name" could refer to both the order and the informally used name. [Wikipedia suggests "given name"](https://en.wikipedia.org/wiki/Given_name) but that doesn't cover all cases either.